app/services/product_converter.py

```
@@ -1,7 +1,6 @@
-# app/services/product_converter.py - 단순화된 버전
 from sqlalchemy.ext.asyncio import AsyncSession
-from sqlalchemy import select
-from typing import List, Tuple
+from sqlalchemy import select, text
```

Ruff (0.11.9)

`sqlalchemy.select` imported but unused

Remove unused import: `sqlalchemy.select`

(F401)